inst/extdata/config.yaml

```diff
@@ -1,58 +1,40 @@
 # This is a config file for the piccr package
 
-FILE_ID: "*.csv"	# file extension used for data files
-COLUMNSEP: ","		# column separator used in data files
-DECSEP: "."   		# decimal separator used in data files
-NAME_StdScheme: IsoLabP_Muench2014.v2  # name of applied standard scheme (optional meta information)
-STD_NO:	5			  # number of standards used in standard scheme
-BLOCK_NO:	3			# number of standard blocks distributed over measurement rack in this standard scheme
```

Comment:
`STD_NO` und `BLOCK_NO` ergeben sich aus den in der config spezifiezierten Standards und den Eingabedaten.

Reply:
I'm Ok with the deletion.